CMSgov/design-system
# [WNMGDS-892] Add outside click utility

## Summary
Overview of changes

### Added
List new features or components. Include a screenshot for new visual elements.

### Changed
List changes in existing functionality or design. 
If the change was visual, include a comparison screenshot showing the before and after the visual change.

## How to test
Instructions on how to test the changes. This is not exhaustive list of ways you should test this PR.

http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-892/add-outside-click-utility/
